projects/plugins/jetpack/extensions/blocks/latex-formula/edit.tsx

```diff
@@ -0,0 +1,167 @@
+import { useBlockProps } from '@wordpress/block-editor';
+import { Button, TextareaControl } from '@wordpress/components';
+import { useState, useEffect, useRef } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import type { BlockEditProps } from '@wordpress/blocks';
+import './editor.scss';
+
+const KATEX_CSS = 'https://cdn.jsdelivr.net/npm/katex@0.16.9/dist/katex.min.css';
+const KATEX_JS = 'https://cdn.jsdelivr.net/npm/katex@0.16.9/dist/katex.min.js';
```

Comment:
Instead of loading the library from a third-party source, I would recommend adding it as a dependency to the Jetpack plugin, like we do for the Slideshow block and the Swiper library:

https://github.com/Automattic/jetpack/blob/2d926e9cfa029ea9b7c93dd89d000854f2c3897a/projects/plugins/jetpack/package.json#L124
https://github.com/Automattic/jetpack/blob/94311072b2e584e620a2e750f00b47a26c7e72d8/projects/plugins/jetpack/extensions/blocks/slideshow/slideshow.php#L237-L243

Reply:
Since the view script will only be enqueued when the block is available? Can't we just do

```js
import ketex from 'ketex`
```